python/spacewalk/server/action/Makefile

```diff
@@ -5,7 +5,7 @@ TOP	= ../..
 SUBDIR	= server/action
 
 SPACEWALK_FILES	= __init__ configfiles kickstart kickstart_guest packages errata hardware reboot \
-          rollback activation script rhnsd rhn_applet virt kickstart_host \
+          rollback activation script rhn_applet virt kickstart_host \
           utils scap image distupgrade clientcert channels
```

Comment:
Looks like `activation`, `scap`, `virt` should be deleted from the list as well. Am I wrong?

Reply:
right! but I'll do the reverse and do the cleanup in the server only once I have removed up2date completely.